frontend/container/main/explore-container.js

```diff
@@ -19,8 +20,15 @@ class Explore extends Component {
 			<div>
 				<ul>
 					{locations.map(location => {
+						console.log(location.id)
+						console.log("link", "" + /country/ + location.id +"")
+
 						return(
-							<li key={location.id}>{location.city}</li>
+							<li key={location.id}>
+								<Link to={"" + /country/ + location.id +""}>
```

Comment:
what are the double strings for, why isnt country wrapped in string .... do you know about template literals.... es6 way of concating data tyoes.